bench/wtperf/runners/checkpoint_stress_schema_ops.wtperf

```diff
@@ -0,0 +1,19 @@
+# A stress configuration, to create long running checkpoints and see how
+# they interfere with schema level operations (table create, drop).
+# Setup a cache size large enough that checkpoints can take a long time.
+conn_config="cache_size=8GB,log=(enabled=false),checkpoint=(wait=30)"
+table_config="leaf_page_max=4k,internal_page_max=16k,type=file"
+icount=10000000
+table_count_idle=100
+# Turn on create/drop of idle tables, and error if a single operation takes
+# more than 5 seconds.
+idle_table_cycle=5
+populate_threads=5
+checkpoint_threads=0
```

Comment:
The other use is for the monitoring output.  It makes note of when checkpoints are running so that they can be noted in the monitor file and correlated to that performance.  I don't think we want to lose that.